twisted/twisted/plugins/tribler_plugin.py

```diff
@@ -41,6 +46,38 @@ def __init__(self):
         self.session = None
         self._stopping = False
 
+    def load_dispersy_communities(self):
```

Comment:
That's basically a copy of stuff that's happening on `tribler_main.py` right? I think it would be better to move it somewhere it can be used from elsewhere and have normal Tribler and the twistd plugin use the same code.